build-docker.gradle

```diff
@@ -25,29 +25,84 @@ buildscript {
 
 apply plugin: com.bmuschko.gradle.docker.DockerRemoteApiPlugin
 
+
 import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
-import org.apache.tools.ant.filters.ReplaceTokens
+import com.bmuschko.gradle.docker.tasks.image.Dockerfile
 
 docker {
-    url.set("${dockerServerUrl}")
+    url.set(dockerServerUrl)
+}
+
+def isRelease = project.hasProperty('registryRelease')
+def devPostfix = isRelease ? '' : '-dev-5'
```

Comment:
lets name the variable as 'imagePostfix' and leave '-dev'

Reply:
done